src/key_value_store/client/kvstore_client.h

```diff
@@ -1,35 +1,41 @@
 #include "../KeyValueStoreInterface.h"
 
-#include <grpcpp/grpcpp.h>
 #include "store.grpc.pb.h"
+#include <grpcpp/grpcpp.h>
 
 #include <string>
 #include <vector>
 
-#include "../core/KeyValueStore.h"
+#include <glog/logging.h>
 
 using grpc::Channel;
 using grpc::ClientContext;
 using grpc::ClientReaderWriter;
 using grpc::Status;
 
-using kvstore::PutRequest;
-using kvstore::PutReply;
-using kvstore::GetRequest;
 using kvstore::GetReply;
-using kvstore::RemoveRequest;
+using kvstore::GetRequest;
+using kvstore::PutReply;
+using kvstore::PutRequest;
 using kvstore::RemoveReply;
+using kvstore::RemoveRequest;
 
+// This class wraps our core Key value store implementation
+// with grpc. It processes a request in cpp syntax input to
+// grpc and process the result to cpp syntax output
 class KeyValueStoreClient final : public KeyValueStoreInterface {
- public:
-
+public:
   KeyValueStoreClient(std::shared_ptr<Channel> channel)
-    : stub_(Greeter::NewStub(channel)) {};
-
+      : stub_(Greeter::NewStub(channel)){};
+  // Performs normal put functionality
```

Comment:
Arguments should be const references.

Also, comments should be more clear / detailed, explaining the arguments and return value.